tutorials/jms-output-exactlyonce/README.md

```diff
@@ -0,0 +1,62 @@
+## JMS ActiveMQ Output Example
+
+
+This application shows the use of JMS output operator by implementing AbstractJMSSinglePortOutputOperator.
+The operator uses its default JMSTransactionalStore to ensure exactly once capability. AbstractJmsOutputOperator
+handles every window as a transaction. In every transaction the current windowId will be saved to a topic specific
+JMS meta-data queue to skip sending tuples of already committed windows.
+It is the responsibility of the user to create the meta-data queue in the JMS provider.
```

Comment:
Is JMS Provider the right term here? I thought JMS Provider refers to the driver library or similar that provides the JMS interface for the actual message broker you are talking to. Pls check